### PR TITLE
Feature/usage policies for vs400

### PR DESCRIPTION
**Pull request by sebbader**
This is the 98% status for the reworked usage policy language. A few things might come tomorrow, but I would like to see already what I have screwed up.

### CODE REVIEW

File: Ontology.ttl [hunk 1]
```diff
@@ -31,16 +31,16 @@ ids:
...
-    dct:modified "2020-04-30"^^xsd:date ;
-    owl:versionInfo "3.1.0" ;
-    owl:versionIRI "https://w3id.org/idsa/core/3.1.0>" ;
+    dct:modified "2020-08-05"^^xsd:date ;
+    owl:versionInfo "4.0.0" ;
+    owl:versionIRI <https://w3id.org/idsa/core/4.0.0> ;
     vann:preferredNamespaceUri "https://w3id.org/idsa/core/" ;
     vann:preferredNamespacePrefix "ids" ;
-    rdfs:seeAlso <https://international-data-spaces-association.github.io/InformationModel/docs/index.html> ;
+    rdfs:seeAlso <https://industrialdataspace.github.io/InformationModel/> ;
```
Comment (JohannesLipp) [inline Ontology.ttl:43]: Why refer to the old URI again? It shows 404...

File: codes/Action.ttl [hunk 2]
```diff
@@ -86,28 +123,31 @@ idsc:READ
     rdfs:comment "To obtain data from the resource."@en ;
     rdfs:label "read"@en ;
     rdfs:seeAlso <http://www.w3.org/ns/odrl/2/read>;
-    skos:note "This action is always evaluated at the provider side. It corresponds to 'give access to a resource'."@en ;	
+    odrl:includedIn idsc:MODIFY ;
```
Comment (JohannesLipp) [inline codes/Action.ttl:126]: Why does "read" imply "modify (change locally)"?

File: codes/ArtifactState.ttl [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# Instances
+# ---------
+
+idsc:ANONYMIZED a ids:ArtifactState;
+    rdfs:label "is anonymized"@en ;
+    rdfs:comment "The artifact is anonymized so the data source can not be recognized by any further user."@en ;
+.
+
+idsc:PSEUDO_ANONYMIZED a ids:ArtifactState;
```
Comment (JohannesLipp) [inline codes/ArtifactState.ttl:24]: I have never heard "pseudo-anonymized" before. Suggestion: Change to "PSEUDONYMIZED"

File: codes/LeftOperand.ttl [hunk 4]
```diff
@@ -12,87 +12,128 @@
...
-    rdfs:comment "Delay the action. Use idsc:DURATION_EQ, idsc:LONGER, idsc:LONGER_EQ, idsc:SHORTER_EQ, or idsc:SHORTER with datatypes xsd:duration."@en ;
-    .
+    rdfs:comment "Delay the action. Use idsc:DURATION_EQ, idsc:LONGER, idsc:LONGER_EQ, idsc:SHORTER_EQ, or idsc:SHORTER with datatype xsd:duration."@en ;
+.
+
+#
+# -----------
+idsc:EVENT a ids:LeftOperand;
+    rdfs:label "current event"@en ;
+    rdfs:comment "TODO"@en ;
```
Comment (JohannesLipp) [inline codes/LeftOperand.ttl:54]: Todo?

File: codes/RequestTemplate.ttl [hunk 5]
```diff
@@ -19,7 +19,7 @@ idsc:RequestTemplate a ids:PolicyTemplate ;
 		Optional: This only holds if <Attribute URI> is [equal|lower than|greater than] the [<Value URI>|<Value Literal>].
 		Optional: This only holds if also <Attribute URI> is [equal|lower than|greater than] the [<Value URI>|<Value Literal>]."""@en ;
     skos:note """{ 
-		"@context": "http://www.w3id.org/ids/contract.jsonld", 
+		"@context": "https://w3id.org/idsa/context.json",
```
Comment (JohannesLipp) [inline codes/RequestTemplate.ttl:22]: "The requested URL was not found on this server." - Will this be resolved by this pull request? If so, then you can ignore this comment

**Comment by jlangkau** [comment]
Hopefully!

**Pull request merged**